eventsdiscovery/src/main/java/com/schedjoules/client/eventsdiscovery/json/JsonEvent.java

```diff
@@ -36,6 +39,7 @@
  */
 public final class JsonEvent implements Event
 {
+    public static final String KEY_DURATION = "duration";
```

Comment:
I agree that it would be good to list the keys on the top as constants but solely for duration, it looks inconsistent for me. (I'd rather have the string literal then.)
`public` modifier left from refactor.